contentcuration/contentcuration/frontend/shared/utils/helpers.js

```diff
@@ -183,10 +183,10 @@ export async function generatePdf(
       const style = window.getComputedStyle(node, null);
       const paddingLeft = parseInt(style.getPropertyValue('padding-left'));
       const marginLeft = parseInt(style.getPropertyValue('margin-left'));
-      const borderLeft = parseInt(style.getPropertyValue('border-left'));
+      const borderLeft = parseInt(style.getPropertyValue('border-left') || '0');
```

Comment:
Maybe worth adding this to everywhere we are doing `parseInt` here just for safety?

Reply:
I just added a new utility `safeParseInt` that returns 0 in these special cases and replaced these calls to `parseInt` with `safeParseInt`